src/extension.ts

```diff
@@ -20,6 +21,21 @@ export function activate(context: vscode.ExtensionContext) {
 
 	let createInstancePanel: CreateInstancePanel | null = null;
 
+	const requireNeoExpress = async (then: Function) => {
+		if (await NeoExpressHelper.isNeoExpressInstalled()) {
+			then();
+		} else {
+			const showInstructions = 'Show installation instructions';
+			const dialogResponse = await vscode.window.showInformationMessage(
+				'Neo Express Required\n\nNeo Express was not detected on your machine. Neo Express must be installed in order to use this functionality.\n',
+				{ modal: true },
+				showInstructions);
+			if (dialogResponse === showInstructions) {
+				vscode.env.openExternal(vscode.Uri.parse('https://github.com/neo-project/neo-express#Installation'));
+			}
+		}
+	};
+
```

Comment:
I like this much better. Should we raise a notification when the install is done? Also, do we continue with the original task after install is complete?

Reply:
we didn't... i just changed it so that we do...